docs/content/latest/releases/earlier-releases/v2.1.5.md

```diff
@@ -0,0 +1,101 @@
+---
+title: What's new in 2.1.5
```

Comment:
remove what's new from this page's front matter

Reply:
Fixed